### ISSUE TITLE
Test check fails (possibly because of old sp package use)

### ISSUE THREAD

**Issue by ifeanyi588** [SSA-Statistical-Team-Projects/povmap#11]
@davidn10, 

I see you made additions which for the most part appear to be fine. But I think it showed an error that might have already been in the system which I imagine EMDI will have to contend with as well. Check this out: 

https://github.com/SSA-Statistical-Team-Projects/povmap/actions/runs/6499221836/job/17652152776

I think the error happens because of the warning message. My guess is that you dont have any errors in R's test check but github throws up errors because github's R Test Check functionality doesn't differentiate between some warnings and outright test check fails. I have put this on my to do list. But we should probably switch to sf because I know R is looking to get rid of some of the package dependencies of sp after October and this will likely mean that EMDI and povmap will no longer be able to make maps if CRAN gets rid of sp and/or its dependencies.   

Ify

**Comment by davidn10** [comment]
Yes I agree we should move to sf so this check passes. I think we currently use sp for drawing maps which is not an essential part of the package. I will add it to my todo list.Best,David On Oct 14, 2023, at 6:11 AM, Ifeanyi Edochie ***@***.***> wrote:﻿
@davidn10,
I see you made additions which for the most part appear to be fine. But I think it showed an error that might have already been in the system which I imagine EMDI will have to contend with as well. Check this out:
https://github.com/SSA-Statistical-Team-Projects/povmap/actions/runs/6499221836/job/17652152776
I think the error happens because of the warning message. My guess is that you dont have any errors in R's test check but github throws up errors because github's R Test Check functionality doesn't differentiate between some warnings and outright test check fails. I have put this on my to do list. But we should probably switch to sf because I know R is looking to get rid of some of the package dependencies of sp after October and this will likely mean that EMDI and povmap will no longer be able to make maps if CRAN gets rid of sp and/or its dependencies.
Ify
